src/derivations/Segmentation.js

```diff
@@ -0,0 +1,278 @@
+    }
+
+    derive() {
+        super.derive();
+
+        this.assignToDataset({
+            SOPClassUID: DicomMetaDictionary.sopClassUIDsByName.Segmentation,
+            Modality: "SEG",
+            SamplesPerPixel: "1",
+            PhotometricInterpretation: "MONOCHROME2",
+            BitsAllocated: "1",
+            BitsStored: "1",
+            HighBit: "0",
+            PixelRepresentation: "0",
+            LossyImageCompression: "00",
+            SegmentationType: "BINARY",
+            ContentLabel: "EXAMPLE"
+        });
+
+        let dimensionUID = DicomMetaDictionary.uid();
+        this.dataset.DimensionOrganizationSequence = {
+            DimensionOrganizationUID: dimensionUID
+        };
+        this.dataset.DimensionIndexSequence = [
+            {
+                DimensionOrganizationUID: dimensionUID,
+                DimensionIndexPointer: 6422539,
+                FunctionalGroupPointer: 6422538, // SegmentIdentificationSequence
+                DimensionDescriptionLabel: "ReferencedSegmentNumber"
+            },
+            {
+                DimensionOrganizationUID: dimensionUID,
+                DimensionIndexPointer: 2097202,
+                FunctionalGroupPointer: 2134291, // PlanePositionSequence
+                DimensionDescriptionLabel: "ImagePositionPatient"
+            }
+        ];
+
+        this.dataset.SegmentSequence = [];
+
+        // TODO: check logic here.
+        // If the referenced dataset itself references a series, then copy.
+        // Otherwise, reference the dataset itself.
+        // This should allow Slicer and others to get the correct original
+        // images when loading Legacy Converted Images, but it's a workaround
+        // that really doesn't belong here.
+        if (this.referencedDataset.ReferencedSeriesSequence) {
+            this.dataset.ReferencedSeriesSequence = DerivedDataset.copyDataset(
+                this.referencedDataset.ReferencedSeriesSequence
+            );
+        } else {
+            const ReferencedInstanceSequence = [];
+
+            for (let i = 0; i < this.referencedDatasets.length; i++) {
+                ReferencedInstanceSequence.push({
+                    ReferencedSOPClassUID: this.referencedDatasets[i]
+                        .SOPClassUID,
+                    ReferencedSOPInstanceUID: this.referencedDatasets[i]
+                        .SOPInstanceUID
+                });
+            }
+
+            this.dataset.ReferencedSeriesSequence = {
+                SeriesInstanceUID: this.referencedDataset.SeriesInstanceUID,
+                StudyInstanceUID: this.referencedDataset.StudyInstanceUID,
+                ReferencedInstanceSequence
+            };
+        }
+
+        if (!this.options.includeSliceSpacing) {
+            // per dciodvfy this should not be included, but dcmqi/Slicer requires it
+            delete this.dataset.SharedFunctionalGroupsSequence
+                .PixelMeasuresSequence.SpacingBetweenSlices;
```

Comment:
[Per standard](http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.7.6.16.2.html#sect_C.7.6.16.2.1), `SpacingBetweenSlices` is 1C, so there must be an error in `dciodvfy`.

But also, dcmqi/Slicer should not require it. I don't see in the code it being required. Can you clarify? Did you mean `SliceThickness`? That one is indeed required, per the same section in the standard as referenced above.

Reply:
This is a question for @pieper (I believe).

Reply:
The code and comment are from a while ago so maybe this can be cleared up.